src/sql/workbench/contrib/notebook/browser/notebookViews/notebookViews.component.html

```diff
@@ -7,9 +7,12 @@
 <div style="overflow: hidden; width: 100%; height: 100%; display: flex; flex-flow: column">
 	<div #viewsToolbar class="editor-toolbar actionbar-container" style="flex: 0 0 auto; display: flex; flex-flow: row; width: 100%; align-items: center;"></div>
 	<div #container class="scrollable" style="flex: 1 1 auto; position: relative; outline: none" (click)="unselectActiveCell()" (scroll)="scrollHandler($event)">
-		<loading-spinner [loading]="isLoading"></loading-spinner>
-		<notebook-views-grid-component #gridstack [views]="views" [cells]="cells" [model]="model"></notebook-views-grid-component>
-		<div class="book-nav" #bookNav [style.visibility]="navigationVisibility">
-		</div>
+		<notebook-views-grid-component #gridstack [views]="views" [activeView]="activeView" [cells]="cells" [model]="model"></notebook-views-grid-component>
+
+		<ng-container *ngFor="let cell of cellsAwaitingInput">
+			<views-modal-component class="modal" [title]="cellsAwaitingInputModalTitle">
```

Comment:
I'm not sure I understand the use of the name "modal" here. Modal dialogs should be a single top-level dialog that requires interaction over everything else. This seems to be just an empty placeholder or something for the views?

Reply:
This is an internal modal for the grid. One example is if a cell is hidden in the View, and the Run All button is clicked, but the cell needs input, we use the modal to show the input field for the cell that is hidden.

Reply:
That's not really a modal - more of just an overlay. A modal is typically a top-level component over the entire app.